@AH-64D Apache Longbow/Addons/fza_ah64_weapons/functions/pylon/fn_PylonSwapper.sqf

```diff
@@ -0,0 +1,67 @@
+/* ----------------------------------------------------------------------------
+Function: fza_weapons_fnc_PylonSwapper
+
+Description:
+	swaps the hellfire and rockets between the pylons
+	
+Parameters:
+	_heli - The helicopter to get information from [Unit].
+
+Returns:
+	...
+
+Examples:
+	...
+
+DEBUG:
+	_heli = vehicle player; 
+    _wassing = "rockets"; 
+    _turret = -1; 
+    _PylonInfo = getAllPylonsInfo vehicle player; 
+    if (vehicle player turretLocal [-1]) then {_turret = 0;}; 
+    [_wassing,_PylonInfo,_heli] remoteExec ["fza_weapons_fnc_pylonswapper", vehicle player turretUnit [_turret]]; 
```

Comment:
I like your design here, how you collect the pylon information, and then send it out to each machine, telling them to match this information.

These 3 lines should be their own function, which we call. Rather than having to duplicate this code in each usage site. That way we have one source of truth. Maybe call it `fza_weapons_fnc_swapPylonOwner`

As a general note for naming of functions. Functions do things, so we name them after the verb (action) that they do, not as a now (an item that does the thing). For example you can take the command `remoteExec` we call it `remoteExec` because it remotely execute codes, we don't call it `remoteExecutor`, same with `getAllPylonsInfo`, it's not called `allPylonsInfoGetter`.